.github/ISSUE_TEMPLATE/release.md

```diff
@@ -179,7 +191,7 @@ in the infrastructure repository. All of these tests can be run in parallel.
 
   - [ ] Ensure that all members of the release-notes team have signed-off on this issue.
 
-  - [ ] Ensure that the announcement blog post has been published and
+  - [ ] For a major release, ensure that the announcement blog post has been published and
     announced, if applicable, by pinging the product team in #release.
```

Comment:
The "if applicable" bit was meant to capture the "for a major release" back when it wasn't as clear that blog posts were published for major releases and major releases only. Can drop the "if applicable" now.

Reply:
Dropped the inf applicable part!